README.md

```diff
@@ -1,6 +1,10 @@
-# fivetwelve driver for ENTTEC USB PRO mk2
+# fivetwelve driver for the USB PRO family by enttec
```

Comment:
I would say "the **DMX** USB PRO family by ENTTEC", because that is what you would search for.